tests/collections/asr/test_jasper_block.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import random
```

Comment:
This is probably not needed since you're using `torch.rand`.

Reply:
I thought of using random.randrange to generate random kernel and dilation sizes, but later found out we cannot always find a padding size for conv1d to make input output same for some even kernel sizes like 4 and dilation 3. 
There must be a reason why pytorch didn't support this to append zeros?

Reply:
We force all kernels to odd size so don't consider even kernel shapes.